file/file-alfresco/src/main/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequester.java

```diff
@@ -48,7 +65,10 @@
         }
     }
 
-    private String alfrescoUriOf(final String fileId, final String fileName) {
+    private String alfrescoUriOf(final String fileId, final String fileName, final boolean transformPdf) {
+        if(transformPdf) {
+            return format("%s%s?transformpdf=%s", alfrescoPdfContentWorkspacePath, fileId, transformPdf);
```

Comment:
I would like to see the Alfresco docs that explain why this PDF URL is different to the standard URL